shared/common-adapters/user-proofs.desktop.js

```diff
@@ -151,6 +151,9 @@ function LoadingProofRow({textBlockWidth}: {textBlockWidth: number}) {
   )
 }
 
+// CSSTransition injects foreign props so lets not just accept all props
+const IgnorePropsBox = ({children, onlyProps}) => <Box {...onlyProps}>{children}</Box>
```

Comment:
This sucks and this whole ux sucks. we should instead have the actual rows do this so it doesn't appear as an overlay. also the loading should be quick enough that we don't need this at all